src/strategies/index.ts

```diff
@@ -460,7 +461,8 @@ const strategies = {
   'path-balance-staked-and-locked': pathBalanceStakedAndLocked,
   'sumami-holders': sUmamiHolders,
   zorro,
-  'volt-voting-power': voltVotingPower
+  'volt-voting-power': voltVotingPower, 
+  'balancer-poolId': balancerPoolId
```

Comment:
Hi @jo-chemla Strategy name should be all lowercase maybe make this `balancer-poolid` or `balancer-pool-id`

Reply:
Thanks, just did change that